src/mono/wasm/debugger/BrowserDebugProxy/ValueTypeClass.cs

```diff
@@ -129,6 +130,12 @@ public async Task<JObject> ToJObject(MonoSDBHelper sdbAgent, bool forDebuggerDis
                 string displayString = await sdbAgent.GetValueFromDebuggerDisplayAttribute(Id, TypeId, token);
                 if (displayString != null)
                     description = displayString;
+                else
+                {
+                    var toString = await sdbAgent.InvokeToStringAsync(new List<int>(){TypeId}, true, IsEnum, Id.Value, IsEnum ? BindingFlags.Default : BindingFlags.DeclaredOnly, token);
```

Comment:
```suggestion
                    var toString = await sdbAgent.InvokeToStringAsync(new int[]{ TypeId }, true, IsEnum, Id.Value, IsEnum ? BindingFlags.Default : BindingFlags.DeclaredOnly, token);
```

.. and named arguments.